shell/platform/darwin/macos/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -523,4 +529,77 @@ + (void)respondFalseForSendEvent:(const FlutterKeyEvent&)event
   }
 }
 
+- (bool)testTrackpadGesturesAreSentToFramework {
```

Comment:
The unit test changes seems underwhelming compared to the implementation added. We probably want to test magify and rotate (probably swipe if possible), and cases of multiple gestures taking place at the same time, `hasPreciseScrollingDeltas`, and resetting. It would be best if they can be roughly based on real recorded event instead of pure mock data.

Reply:
Upon a little research it does not seem to be possible without basically hex-editing a CGEvent to create synthetic magnification or rotation events.